diff --git a/biome_fmt.js b/biome_fmt.js
index 0df459556c6a5dca8e4704531441d1747fe50a07..778a7c9867f64ffce517091e74b972769827a8b3 100644
--- a/biome_fmt.js
+++ b/biome_fmt.js
@@ -431,18 +431,12 @@ async function __wbg_init(input) {
     }
     const imports = __wbg_get_imports();
 
-    switch (true) {
-        case typeof input === 'string':
-            input = new URL(input, import.meta.url);
-        case input instanceof URL:
-            if (typeof __webpack_require__ !== 'function' && input.protocol === 'file:') {
-                const fs = await import('node:fs/promises');
-                input = fs.readFile(input);
-                break;
-            }
-        case typeof Request === 'function' && input instanceof Request:
-            input = fetch(input);
-    }
+	if (
+		typeof input === "string" || (typeof Request === "function" && input instanceof Request)
+		|| (typeof URL === "function" && input instanceof URL)
+	) {
+		input = fetch(input);
+	}
 
     __wbg_init_memory(imports);
 
